rag-langchain-python/vector_query_retriever.py

```diff
@@ -56,9 +58,10 @@ def _build_eval_script(self, query, query_embedding):
     def _get_relevant_documents(self, query: str) -> List[Document]:
         query_embedding = self.embedding_generator.embed_query(query)
         eval_script = self._build_eval_script(query, query_embedding)
-        results = self.client.eval(javascript=eval_script)
+        optic_rows = self.client.eval(javascript=eval_script)
+        print(optic_rows[1].keys())
 
-        print(f"Count of MarkLogic documents sent to the LLM: {len(results)}")
-        for result in results:
-            print(f"URI: {result['uri']}")
-        return map(lambda doc: Document(page_content=doc["text"]), results)
+        print(f"Count of MarkLogic documents sent to the LLM: {len(optic_rows)}")
```

Comment:
Should change the message to be "MarkLogic rows", or maybe even "MarkLogic chunks"?